src/models/section.py

```diff
@@ -1,3 +1,14 @@
 from remodel.models import Model
+from remodel.helpers import create_tables, create_indexes
+
 class Section(Model):
     belongs_to = ('Course',)
```

Comment:
@antsankov we will add belongs_to / has_many relationships when we begin working on those components. Until then, `instructor.py` doesn't exist yet.